portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase

```diff
@@ -490,4 +490,22 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-123: After the administrator logs in, create a new administrator user and log in for verification"
```

Comment:
Your steps are right, just a suggestion, you'd better simplify the description, such as: Assert we can add and log in with the new administrator user successfully.

Reply:
OK, I  will describe in short language next time，thank you!